spec/lib/rbac/filterer_spec.rb

```diff
@@ -1147,20 +1147,31 @@ def combine_filtered_ids(user_filtered_ids, belongsto_filtered_ids, managed_filt
           :max_disk_usage_rate_average     => {},
           :min_net_usage_rate_average      => {},
           :max_net_usage_rate_average      => {},
-          :v_derived_storage_used          => {}
+          :v_derived_storage_used          => {},
+          :resource                        => {}
         }
       end
 
       # NOTE:  Think long and hard before you consider removing this test.
       # Many-a-hours wasted here determining this bug that resulted in
       # re-adding this test again.
+      #
+      # 2nd NOTE:  I did think (and wrote the above comment as well), however,
+      # now it seems we DO NOT SUPPORT polymorphic code, since we removed it
+      # here:
+      #
+      #   https://github.com/ManageIQ/manageiq/commit/8cc2277b
+      #
+      # That said, we should make sure this is erroring and no other callers
+      # are trying to do this (example:  MiqReport), so make sure this raises
+      # an error to inform the caller that fixing needs to happen.
+      #
       it "should not raise an error when a polymorphic reflection is included" do
-        result = nil
         expect do
           result = described_class.search :class            => "MetricRollup",
-                                          :include_for_find => @include
-        end.not_to raise_error
-        expect(result.first.length).to eq(1)
+                                          :include_for_find => @include,
+                                          :references       => @include
+        end.to raise_error(Rbac::PolymorphicError)
```

Comment:
`include_for_find` (aka `includes()` is fine but `references` (aka `references()`) is not. Right?

Reply:
Pretty sure I ripped this off from something in your PR and the specs there... looking...

Reply:
aah - I was saying that if you just pass `include_for_find` without the `references` then that should work. But passing in the `references` should cause it to fail.

Can we do both tests?

Reply:
Sure.